### PR TITLE
fix: only support node 6 and above

### PR DESCRIPTION
**Pull request by egoist**
Fixed https://github.com/babel-utils/babel-plugin-tester/issues/15

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "presets": [
     ["env", {
       "targets": {
-        "node": "4.5"
+        "node": 6
```
Comment (kentcdodds) [inline .babelrc]: Hmmm... I'm unsure how this helps resolve the issue... Also, I seem to remember seeing log messages from babel stating that this needs to be a string.

Comment (egoist) [inline .babelrc]: it will transform async function to generator instead, without regeneratorRuntime.

Comment (egoist) [inline .babelrc]: So I changed it to exclude `transform-regenerator` instead, this way we won't need to modify tests.

**Comment by kentcdodds** [comment]
Oh, now I understand. Yes, this will work great. Thank you very much for clarifying and providing the fix!

**Pull request merged**